src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/NukleusWriteScheduler.java

```diff
@@ -292,7 +310,7 @@ private MutableDirectBuffer acquireReplyBuffer(UnaryOperator<MutableDirectBuffer
             if (replySlot != NO_SLOT)
             {
                 int capacity = slab.buffer(replySlot).capacity();
-                replyBuffer = new CircularDirectBuffer(capacity);
+                replyBuffer = new CircularEntryBuffer(capacity);
                 replyQueue = new LinkedList<>();
```

Comment:
We'll need to do another pass on this design (later) to analyze memory usage and GC.